### PR TITLE
Create answer.txt

### PR DESCRIPTION
**Pull request by danielbr08**

### CODE REVIEW

File: modules/01 JavaScript Programming/Assignments/8/answer.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var bloodPressure = 70;
+var isNormal = false;
+isNormal = (bloodPressure > 80 && bloodPressure < 120) ? true : false; 
+
+console.log(isNormal);
+
+isNormal = (bloodPressure < 80 || bloodPressure > 120) ? true : false;
```
Comment (JavascriptBootcamp) [inline modules/01 JavaScript Programming/Assignments/8/answer.txt]: you should've assigned the opposite result

File: modules/01 JavaScript Programming/Assignments/8/answer.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var bloodPressure = 70;
+var isNormal = false;
+isNormal = (bloodPressure > 80 && bloodPressure < 120) ? true : false;
```
Comment (JavascriptBootcamp) [inline modules/01 JavaScript Programming/Assignments/8/answer.txt]: "between" means >=80 && <=120

Comment (danielbr08) [reply modules/01 JavaScript Programming/Assignments/8/answer.txt]: I sent a correction ..

File: modules/01 JavaScript Programming/Assignments/8/answer.txt (hunk 1 above)
Comment (danielbr08) [reply modules/01 JavaScript Programming/Assignments/8/answer.txt]: I sent a correction ..

**Pull request closed without merging**